pkg/maps/lbmap/lbmap.go.orig

```diff
@@ -0,0 +1,606 @@
+// Copyright 2016-2019 Authors of Cilium
```

Comment:
Looks like this was maybe committed by mistake?

Reply:
:facepalm: